test/end-to-end/pages/language/bilingualgrant.js

```diff
@@ -8,8 +8,14 @@ module.exports = async function(language = 'en', answer) {
     const commonContent = language === 'en' ? contentEn : contentCy;
 
     await I.checkPageUrl('app/steps/ui/language');
-    const locator = {css: `#bilingual${answer}`};
-    await I.waitForEnabled(locator);
+    let locator = '';
+    if (answer === '-2') {
+        locator = {xpath: '//*[@type="radio" and @value="optionNo"]'};
+    } else {
+        locator = {xpath: '//*[@type="radio" and @value="optionYes"]'};
+    }
+    await I.wait(20);
```

Comment:
Sorry but we need explicit waits not implicit, spent a long time erradicating these